manage.py

```diff
@@ -6,7 +6,7 @@
     if 'test' in sys.argv:
         os.environ['DJANGO_SETTINGS_MODULE'] = 'icommons_ext_tools.settings.test_settings'
     else:
-        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'icommons_ext_tools.settings.local')
+        os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'icommons_ext_tools.settings.aws')
```

Comment:
Yeah, my major worry here was having to remember to pass it all the time when running under vagrant.